packages/block-editor/src/components/block-preview/dropdown-wrapper.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * BlockPreviewDropdownWrapper provides styling so you can show Preview inside
+ * Dropdown components. It expects BlockPreview as a single child.
+ *
+ * @param {Object} props Component props.
+ *
+ * @return {WPElement} Rendered element.
+ */
+export default function BlockPreviewDropdownWrapper( { children } ) {
```

Comment:
Do we really need this component, or framed differently, how is this related to "preview", it seems like a component used to display something at the right of a popover, right? Should be named something like `PopoverAside` or something like that?

Reply:
> Should be named something like `PopoverAside` or something like that?

In this case, are you suggesting to put it out of the `<BlockPreview />` component?

Maybe we could try to get the chance to increase its functionality such as `anchorTo: 'top-right`, or even more, make it `autoLocate`: `true`?

For instance, depending on the viewport width the dropdown isn't shown:

![image](https://user-images.githubusercontent.com/77539/62294396-32394a00-b441-11e9-8887-258b9a3c7304.png)

**It's important to clarify that this wasn't introduced in this PR. It already happens in prod.**

However, creating a new `<PopoverAside />` is out of the scope of this patch imho.

Reply:
This PR extracted a new component called `BlockPreviewDropdownWrapper` but if you see its code, there's nothing preview specific aside the label maybe? I agree that it's fine to keep this way. I personally this component is not really needed and should probably be removed and just copy the "preview" title where we need it.

Especially with things like #16813 where this component won't be needed in the inserter anymore.

Reply:
> This PR extracted a new component called BlockPreviewDropdownWrapper but if you see its code, there's nothing preview specific aside the label maybe?

For this reason, is what I thought it should be handled by `<BlockPreview />` component.

> I personally this component is not really needed and should probably be removed and just copy the "preview" title where we need it.

... and anchoring the `<PreviewBlock />` component as it currently does? For instance, the `<BlockSwitcher />` takes over this applying styles to the `.block-editor-block-preview` class.